ipa-core/src/cli/verbosity.rs

```diff
@@ -20,6 +24,11 @@ pub struct Verbosity {
     /// Verbose mode (-v, or -vv for even more verbose)
     #[arg(short, long, action = clap::ArgAction::Count, global = true)]
     verbose: u8,
+
+    /// This option is mutually exclusive with console logging. If set,
```

Comment:
can we have both? K8s logs are easy to pull for many shards whereas going into each host is a hassle